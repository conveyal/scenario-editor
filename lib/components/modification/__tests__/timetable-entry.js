/* global describe, it, expect, jest */

describe('Component > TimetableEntry', () => {
  const renderer = require('react-test-renderer')
  const React = require('react')
  const {mockTimetable} = require('../../../utils/mock-data')
  const TimetableEntry = require('../timetable-entry')

  it('renders correctly', () => {
    const update = jest.fn()
    const tree = renderer.create(
      <TimetableEntry
        timetable={mockTimetable}
        update={update}
        />
    ).toJSON()
    expect(tree).toMatchSnapshot()
    expect(update).not.toBeCalled()
  })
})
