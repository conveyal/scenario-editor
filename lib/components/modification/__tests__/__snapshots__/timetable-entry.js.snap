exports[`Component > TimetableEntry renders correctly 1`] = `
<div>
  <div
    className="form-inline">
    <div
      className="checkbox">
      <label>
        <input
          checked={true}
          onChange={[Function]}
          type="checkbox" />
         
        Mon
      </label>
    </div>
    <div
      className="checkbox">
      <label>
        <input
          checked={true}
          onChange={[Function]}
          type="checkbox" />
         
        Tue
      </label>
    </div>
    <div
      className="checkbox">
      <label>
        <input
          checked={true}
          onChange={[Function]}
          type="checkbox" />
         
        Wed
      </label>
    </div>
    <div
      className="checkbox">
      <label>
        <input
          checked={true}
          onChange={[Function]}
          type="checkbox" />
         
        Thu
      </label>
    </div>
    <div
      className="checkbox">
      <label>
        <input
          checked={true}
          onChange={[Function]}
          type="checkbox" />
         
        Fri
      </label>
    </div>
    <div
      className="checkbox">
      <label>
        <input
          checked={false}
          onChange={[Function]}
          type="checkbox" />
         
        Sat
      </label>
    </div>
    <div
      className="checkbox">
      <label>
        <input
          checked={false}
          onChange={[Function]}
          type="checkbox" />
         
        Sun
      </label>
    </div>
  </div>
  <div
    className="form-group">
    <label>
      Frequency (minutes)
    </label>
    <input
      className="form-control"
      name={undefined}
      onChange={[Function]}
      placeholder="Frequency (minutes)"
      type="number"
      value={15} />
  </div>
  <div
    className="form-group">
    <label>
      Start time
    </label>
    <input
      className="form-control"
      name={undefined}
      onChange={[Function]}
      placeholder="Start time"
      type="time"
      value="08:00" />
  </div>
  <div
    className="form-group">
    <label>
      End time
    </label>
    <input
      className="form-control"
      name={undefined}
      onChange={[Function]}
      placeholder="End time"
      type="time"
      value="16:00" />
  </div>
</div>
`;
